classifier/models.py

```diff
@@ -20,6 +21,6 @@ class Category(models.Model):
     news = models.ForeignKey(News, on_delete=models.CASCADE)
     category = models.CharField(max_length=100)
     prob_score = models.FloatField(default=0)
-
+    ## lime viz here 
```

Comment:
I agree. We probably need to save the html snippet into a field in the SQL. maybe name it explainer?  
``` explainer = models.xxxxxx ```

Can you take a look at what data types allows store strings of this size? It's gonna be long str text

Reply:
Actually, while looking for data types that can store longer string. We should switch the news main_text to the same category. Then It would allow user to input a much longer passage.

Reply:
When we modified the models.py file, a lot of configuration will have changed. 

Take a look at https://docs.djangoproject.com/en/3.0/intro/tutorial02/

In particular, the part related to ```python manage.py sqlmigrate polls 0001```